frontend/src/metabase/forms/components/FormGroupsWidget/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./FormGroupWidget";
```

Comment:
I think the `s` got left out of the filename, even though it’s in the component name.  Can you rename the file?

```suggestion
export * from "./FormGroupsWidget";
```